### PR TITLE
Add new view of libaries vulnerabilities

### PR DESCRIPTION
**Pull request by adudekyeti**
image
image

### CODE REVIEW

**Comment by mariuszkrzaczkowski** [comment]
## Migration Recommended

It appears you are still using the CoPilot OAuth App - we recommend you [migrate to the GitHub App](https://github.com/apps/blackduck-copilot/installations/new/permissions?suggested_target_id=10201412&repository_ids[]=28060246)

**Comment by mariuszkrzaczkowski** [comment]
## Migration Recommended

It appears you are still using the CoPilot OAuth App - we recommend you [migrate to the GitHub App](https://github.com/apps/blackduck-copilot/installations/new/permissions?suggested_target_id=10201412&repository_ids[]=28060246)

**Comment by mariuszkrzaczkowski** [comment]
## Black Duck Security Report

Merging #11278 into developer will not change security risk.

[Click here to see full report](https://copilot.blackducksoftware.com/github/repos/YetiForceCompany/YetiForceCRM/pull-requests/11278)

File: modules/Settings/YetiForce/views/Vulnerabilities.php [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+	/**
+	 * Process user request.
+	 *
+	 * @param \App\Request $request
+	 */
+	public function process(\App\Request $request)
+	{
+		$viewer = $this->getViewer($request);
+		$qualifiedModuleName = $request->getModule(false);
+		$viewer->assign('MODULE', $qualifiedModuleName);
```
Comment (rskrzypczak) [inline modules/Settings/YetiForce/views/Vulnerabilities.php]: ?

File: app/Utils/Dependency.php [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+ * @license   YetiForce Public License 3.0 (licenses/LicenseEN.txt or yetiforce.com)
+ * @author    Arkadiusz Dudek <a.dudek@yetiforce.com>
+ */
+
+namespace App\Utils;
+
+/**
+ * Dependency .
+ */
+class Dependency
```
Comment (rskrzypczak) [inline app/Utils/Dependency.php]: \App\Security\Dependency() vs \App\Utils\Dependency() ?

File: app/Utils/Dependencies.php [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+ */
+
+namespace App\Utils;
+
+/**
+ * Dependencies.
+ */
+class Dependencies
+{
+	private $checkUrl = 'https://security.symfony.com/check_lock';
```
Comment (mariuszkrzaczkowski) [inline app/Utils/Dependencies.php]: doc

File: app/Utils/Dependencies.php [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+	/**
+	 * Send lock file to verify.
+	 *
+	 * @return array
+	 */
+	public function check(): array
+	{
+		$result = [];
+		if (\App\RequestUtil::isNetConnection() && !empty($lockFile = $this->getLockFile(ROOT_DIRECTORY))) {
+			$headers = \App\RequestHttp::getOptions();
```
Comment (mariuszkrzaczkowski) [inline app/Utils/Dependencies.php]:
```suggestion
			$pptions = \App\RequestHttp::getOptions();
```

**Pull request merged**